app-web/plugins/gatsby-remark-path-transform/index.js

```diff
@@ -28,27 +29,54 @@ const { isRelativePath } = require('./utils/utils');
  * @param {Object} options 
  */
 const transformRelativePaths = ({ markdownAST, markdownNode, getNode }, { converter } = {}) => {
+  const IMAGE = 'image';
+  const LINK = 'link';
+
   if (!converter || !TypeCheck.isFunction(converter)) {
     throw new Error(
       "gatsby-remark-path-transform option: 'converter' must be passed in as a function!",
     );
   }
-  const parentQLNode = getNode(markdownNode.parent);
-  // visit anchor tags and images
-  visit(markdownAST, 'image', node => {
+
+  /**
+   * Closured call back to call that returns the url transformed by the 
+   * converter cb or the url on its own if it isn't relative
+   * @param {String} nodeType 
+   * @param {Object} parentQLNode 
+   */
+  const visitCB = (nodeType, parentQLNode) => url => {
```

Comment:
Is this needed to be inside transformRelativePaths?

Reply:
I suppose it doesn't, its a helper callback for the visit fn, would you suggest placing it outside so that it is testable?